package.json

```diff
@@ -12,22 +12,23 @@
   },
   "main": "./client/dist/extension.js",
   "scripts": {
-    "coverage": "nyc --exclude ./src/test npm run test",
+    "clean": "rimraf *.vsix ./coverage/ ./.nyc_output/ ./client/dist/ ./client/out/ ./server/dist/ ./server/out/",
     "cypress:open": "cypress open",
-    "vscode:prepublish": "npm run _webpack_ -- --mode production",
-    "_webpack_": "webpack --stats=errors-warnings --config ./client/webpack.config.js --config ./server/webpack.config.js",
-    "webpack": "npm run _webpack_ -- --mode development",
-    "webpack:watch": "npm run _webpack_ -- --mode development --watch",
     "package": "vsce package",
-    "build": "tsc --build",
-    "watch": "npm run build -- -watch",
+    "build:ts": "tsc --build",
+    "build:webpack": "webpack --stats=errors-warnings --mode development --config ./client/webpack.config.js --config ./server/webpack.config.js",
```

Comment:
I know it's maybe a bit out of scope, but could we go with `--stats=minimal`? I've found it super helpful to know when the errors that show are from the latest watched build or not